TABTestKit/Classes/BDD/Step.swift

```diff
@@ -21,83 +16,40 @@ public typealias And = Step
 ///
 /// Do not use this Step type directly, instead use one of the typealiases above, like Given, When, Then or And.
 /// To learn more about how to use a Step read this article: https://medium.com/kinandcartacreated/swifty-gherkins-part-1-28abba7dfd8
-public struct Step {
+open class Step {
 	
 	/// A reference to the most recently created Step, useful if you want to find out what step failed.
 	public static var current: Step?
-	/// The line in a test function this step is in.
+	/// The test case this step is executed in.
+    public let testCase: XCTestCase
```

Comment:
This probably doesn't need to be here either, since you could add a static property onto TABTestCase and assign the currently executing test case to it in setUp, then reference it where needed like `TABTestCase.currentTestCase`

Reply:
I'm not sure if that's a good idea, because it limits you to execute your tests on one device

Reply:
It doesn't, when you run tests in parallel on multiple sims they run in separate processes, so they'll reference different TABTestCase.currentTestCase values

Reply:
Otherwise we'd have the same issue with the `Step.current` :)

Reply:
Processed feedback 👍